www/js/index.js

```diff
@@ -38,13 +38,6 @@ var app = {
         document.getElementById( "projectId" ).value = projectId;
         document.getElementById( "destinationId" ).value = destinationId;
         document.getElementById( "orderId" ).value = orderId;
-
-        // Request Location When in Use Authorization
-        cordova.plugins.diagnostic.requestLocationAuthorization(
```

Comment:
Have you tested this change on iOS?

Reply:
Yes, Location Permission is asked from `bdFunctions.js`  for `iOS`